lib/model/rwfolder.go

```diff
@@ -780,13 +778,9 @@ func (f *rwFolder) renameFile(source, target protocol.FileInfo) {
 	from := filepath.Join(f.dir, source.Name)
 	to := filepath.Join(f.dir, target.Name)
 
-	if f.versioner != nil {
```

Comment:
Probably e cleanest would be to change the versioner API to take full responsibility for committing the new file, i.e., methods Replace(a, b), Remove(f), Rename(a, b). That's a slightly bigger change, and for v0.14